tensilelite/Tensile/bin/TensileCreateLibrary.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+################################################################################
+#
+# Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+################################################################################
+
+relative2absolute()
+{
+    local relative_path="$1"
+    local absolute_path
+    cd "$(dirname "$relative_path")" || exit
+    absolute_path=$(pwd)/$(basename "$relative_path")
+    if ! cd - > /dev/null; then
+        echo "Error: Failed to return to the previous directory" >&2
+        return 1
+    fi
+    echo $absolute_path
+}
+
+ROCM_PATH="${ROCM_PATH}"
+if [ -z "$ROCM_PATH" ]; then
+    ROCM_PATH="/opt/rocm"
+fi
+WORKING_DIR=/tmp
+ROCISA_LIB_DIR=$(relative2absolute $WORKING_DIR)/rocisa/lib
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+ROCISA_DIR=$SCRIPT_DIR/../../rocisa
+### Build rocisa
+# Currently the path to rocisa is fixed
+mkdir -p $WORKING_DIR/rocisa
+return_code=$?
```

Comment:
If you use `set -e` mode, the script will fail if any command fails, which is usually what you want and can avoid explicit handling.

Reply:
Fixed